generators/Generators.csproj

```diff
@@ -15,4 +15,9 @@
   <ItemGroup>
     <EmbeddedResource Include="Output\Templates\*.liquid" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Hm.. on my IDE I was getting a big red x next to the lib/ directory as well as seeing the lib2git DLL. You can see my issue here, but unfortunately the opener just closed it without much resolve.

https://github.com/libgit2/libgit2sharp/issues/1425